submissions/ArthurGorbenko/HTML5-Form/index.html

```diff
@@ -0,0 +1,487 @@
+<html lang="en">
+  <head>
+    <link
+      href="https://fonts.googleapis.com/css?family=Open+Sans&display=swap"
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" href="node_modules/normalize.css/normalize.css" />
+    <link rel="stylesheet" href="css/styles.css" />
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <title>Document</title>
+  </head>
+  <body>
+    <header class="page_header">
+      <button type="button" class="button page_header_button_back">
+        <h4 class="page_header_button_back_text">Go back</h4>
+      </button>
+      <img src="images/logo-header.png" alt="logo" class="header_logo" />
+      <input
+        type="button"
+        value="Sign in"
+        class="button page_header_button_forward"
+      />
+    </header>
+    <main>
+      <form action="#" class="form">
+        <section class="form-header">
+          <section class="form-header_status">
+            <div class="wrapper_form-header_status_content">
+              <h4 class="form-header_titles form-header_status_title_text">
+                Your order
+              </h4>
+              <table class="form-header_info_table">
+                <thead>
+                  <th class="form-header_status_table_cell">ITEMS</th>
```

Comment:
Avoid writing in uppercase. Text can be changed but the visual style must be the same. For visual presentation use CSS.

Reply:
Done.